xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -243,7 +260,7 @@ glusterd_handle_unfriend_req(rpcsvc_request_t *req, uuid_t uuid, char *hostname,
 
     if (peerinfo == NULL) {
         RCU_READ_UNLOCK;
-        gf_msg("glusterd", GF_LOG_CRITICAL, 0, GD_MSG_REQ_FROM_UNKNOWN_PEER,
+        gf_msg(this->name, GF_LOG_CRITICAL, 0, GD_MSG_REQ_FROM_UNKNOWN_PEER,
```

Comment:
I'm unsure why we need this change (and therefore the changes added in lines 248-251).